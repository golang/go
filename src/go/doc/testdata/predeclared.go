// Copyright 2016 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Package predeclared is a go/doc test for handling of
// exported methods on locally-defined predeclared types.
// See issue 9860.
package predeclared

type error struct{}

// Must not be visible.
func (e error) Error() string {
	return ""
}

type bool int

// Must not be visible.
func (b bool) String() string {
	return ""
}
