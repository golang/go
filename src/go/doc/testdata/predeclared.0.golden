// Package predeclared is a go/doc test for handling of exported ...
PACKAGE predeclared

IMPORTPATH
	testdata/predeclared

FILENAMES
	testdata/predeclared.go
